src/main/java/uk/gov/hmcts/reform/userprofileapi/service/impl/ValidationHelperServiceImpl.java

```diff
@@ -60,20 +61,25 @@ public void validateUserIsPresent(Optional<UserProfile> userProfile) {
         }
     }
 
-    public boolean validateUpdateUserProfileRequestValid(UpdateUserProfileData updateUserProfileData, String userId, ResponseSource source) {
+    public boolean validateUpdateUserProfileRequestValid(UpdateUserProfileData updateUserProfileData, String userId,
+                                                         ResponseSource source) {
         if (!validateUserProfileStatus(updateUserProfileData)) {
             auditService.persistAudit(BAD_REQUEST, source);
-            final String exceptionMsg = String.format("RequiredFieldMissingException - Update user profile request has invalid status %s for userId: %s", updateUserProfileData.getIdamStatus(), userId);
+            final String exceptionMsg = String.format("RequiredFieldMissingException - Update user profile request has "
```

Comment:
I think this is wrong no need to update concat in format. It fails keep as is  

            final String exceptionMsg = String.format("RequiredFieldMissingException - Update user profile request has invalid status %s for userId: %s", updateUserProfileData.getIdamStatus(), userId);